src/PartyMenu/PartyMenu.css

```diff
@@ -0,0 +1,32 @@
+
+
+.party-menu {
+  font-family: 'PkmnFont';
+  text-align: center;
+  background-color: #1e3136;
+  min-height: 100vh;
+  max-height: 100vh;
+  /* min-width: 100vmax; */
+  max-width: 100vw;
+  color: white;  align-items: center;
+  align-self: center;
+
+  background-size: contain;
+
+  a:link {
```

Comment:
For developer empath, it may be good to rename "a" to something descriptive. Unsure what this does at a glance.